File: apps/api-dotnet/src/Planora.Application/Features/Parcels/Dtos/ParcelDetail/ParcelDetailResponse.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Planora.Application.Features.Parcels.Dtos.ParcelDetail
{
    public sealed record ParcelDetailResponse(
        Guid Id,
        string Name,
        decimal AreaHectares,
        string? Country,
        string? Governorate,
        string Status,
        string? GeojsonKey,
        DateTime CreatedAt);
}

File: apps/frontend/src/app/features/parcels/interfaces/parcel-detail/parcel-detail-response.ts
export interface BoundaryPoint {
  longitude: number;
  latitude: number;
}

export interface ParcelDetailResponse {
  id: string;
  name: string;
  areaHectares: number;
  country: string | null;
  governorate: string | null;
  status: string;
  geojsonKey: string | null;
  createdAt: string;
  centroidLatitude: number;
  centroidLongitude: number;
  boundaryCoordinates: BoundaryPoint[];
}
